.travis.yml

```diff
@@ -165,9 +165,10 @@ before_install:
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
 #Fix a broken homebrew libtool install
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew reinstall -s libtool; fi
- - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ccache cppunit bison flex protobuf@3.1 libftdi libftdi0 libusb liblo libmicrohttpd; fi # ossp-uuid already present
+ - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ccache bison flex protobuf@3.1 libftdi libftdi0 libusb liblo libmicrohttpd; fi # ossp-uuid already present
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew link -f protobuf@3.1; export PKG_CONFIG_PATH=/usr/local/opt/protobuf@3.1/lib/pkgconfig; brew install --build-from-source --ignore-dependencies --env=std protobuf-c; fi # While protobuf is not on the latest release
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then mkdir -p ${HOME}/Library/Python/2.7/lib/python/site-packages; echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> ${HOME}/Library/Python/2.7/lib/python/site-packages/homebrew.pth; fi
+ - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then orig_dir=`pwd`; cd $(brew --repo homebrew/core); git checkout e6e43cf6a3^ -- Formula/cppunit.rb; brew install cppunit; cd $orig_dir; fi # install a slightly older cppunit, as latest needs C++11 support
```

Comment:
You mean like https://github.com/OpenLightingProject/ola/pull/1247/commits/12142f73d56553e417474d6374a95b6bc40d648d ?

When it failed ( https://travis-ci.org/peternewman/ola/jobs/229452957#L453 ) I assumed git was too old, I'm not used to the order of flags mattering.